windows/FileUtils.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.Storage;
+
+namespace CodePush.ReactNative
+{
+    class FileUtils
+    {
+        public async static Task MergeDirectories(StorageFolder source, StorageFolder target)
```

Comment:
I think we should replace "Directories" with "Folders" since that is the common term in UWP.